#!/usr/bin/env python
PACKAGE = "explorationISL"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("max_velocity", double_t, 0, "Max Velocity parameter", 0.2, 0.1, 1.0)
gen.add("translation_interval", double_t, 0, "Translation interval in meters", 1.0, 0.2, 10.0)
gen.add("active", bool_t, 0, "ON/OFF autonomous exploration", False)


exit(gen.generate(PACKAGE, "explorationISL", "ExplorationISL"))
